raiden/tests/integration/network/proxies/test_payment_channel.py

```diff
@@ -1,195 +1,50 @@
 import pytest
-from eth_utils import decode_hex, encode_hex, to_canonical_address, to_checksum_address
 
-from raiden.constants import EMPTY_HASH
-from raiden.exceptions import ChannelOutdatedError
+from raiden.constants import EMPTY_HASH, EMPTY_SIGNATURE
+from raiden.exceptions import (
+    ChannelOutdatedError,
+    RaidenRecoverableError,
+    RaidenUnrecoverableError,
+)
 from raiden.network.blockchain_service import BlockChainService
-from raiden.network.proxies.payment_channel import PaymentChannel
-from raiden.network.proxies.token_network import TokenNetwork
 from raiden.network.rpc.client import JSONRPCClient
-from raiden.tests.integration.network.proxies import BalanceProof
+from raiden.transfer.identifiers import CanonicalIdentifier
 from raiden.utils import privatekey_to_address
-from raiden.utils.signer import LocalSigner
 from raiden_contracts.constants import TEST_SETTLE_TIMEOUT_MIN
 
 
 def test_payment_channel_proxy_basics(
```

Comment:
`test_payment_channel_proxy_basics` and `test_payment_channel_outdated_channel_close` were pretty much equal, the only difference is that `basics` was doing a deposit and `outdated` was opening a second channel, everything else was the same. So I improved the assert messages and merged the two.

Reply:
I also changed the test to use the payment channel proxy instead of the token network proxy, otherwise this test doesn't make sense.